src/test/java/framework/DarkSkyLandingPage.java

```diff
@@ -0,0 +1,114 @@
+package framework;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import ru.yandex.qatools.allure.annotations.Step;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+public class DarkSkyLandingPage extends BasePage {
+
+    private By twoHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[3]");
+    private By fourHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[5]");
+    private By sixHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[7]");
+    private By eightHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[9]");
+    private By tenHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[11]");
+    private By twelveHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[13]");
+    private By fourteenHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[15]");
+    private By sixteenHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[17]");
+    private By eighteenHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[19]");
+    private By twentyHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[21]");
+    private By twentytwoHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[23]");
+
+    private By currentTemp = By.xpath("//span[@class='summary swap']");
+    private By lowTemp = By.xpath("//span[@class='currently']//span[2]//span[2]//span[2]");
+    private By highTemp = By.xpath("//span[@class='summary-high-low']//span[3]");
+
+
+//https://dzone.com/articles/getting-current-date-time-in-java
+//https://stackoverflow.com/questio
+
+    @Step
+    public void getCurrentTime() {
```

Comment:
This should not in page object, you should create another class and store it there